src/Simulation/CsharpGeneration/SimulationCode.fs

```diff
@@ -274,7 +274,7 @@ module SimulationCode =
         override this.OnStatement (node:QsStatement) =
             match node.Location with
             | Value loc ->
-                let (current, _) = loc.Offset
+                let current = loc.Offset.Line
                 parent.LineNumber <- parent.StartLine |> Option.map (fun start -> start + current + 1) // The Q# compiler reports 0-based line numbers.
```

Comment:
Is the `+ 1` here still necessary?

Reply:
Yes, I think so, since `LineNumber` is one-based.